<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-0F8J9IN" ignored="0" name="Test" tests="2" failures="1" timestamp="2021-02-17T22:56:14 EST" time="16.152" errors="0">
  <testcase name="TestLogin" time="1.669" classname="testCases.TC_OWA_001"/>
  <testcase name="TestLogin" time="0.867" classname="testCases.TC_OWA_001">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [true] but found [false]
at testCases.TC_OWA_001.TestLogin(TC_OWA_001.java:40)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- TestLogin -->
</testsuite> <!-- Test -->
